amoro-ams/pom.xml

```diff
@@ -408,6 +409,66 @@
             <scope>test</scope>
         </dependency>
 
+        <dependency>
```

Comment:
we can use ${swagger.version} replace 2.2.10

Reply:
Yes, we may define the versions in properties.